services/frontend/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.7'
+
+x-frontend: &frontend
+  build:
+    context: .
+    dockerfile: services/frontend/Dockerfile
+  image: frontend
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+  working_dir: /govuk/frontend
+
+services:
+  frontend-lite:
+    <<: *frontend
+    privileged: true
+    volumes:
+      - ..:/govuk:delegated
+      - bundle:/usr/local/bundle
+      - ./frontend/chromedriver-version:/home/build/.chromedriver-helper/.chromedriver-version
+
+  frontend-app: &frontend-app
+    <<: *frontend
+    depends_on:
+      - content-store-app
+      - router-app
+      - static-app
+    environment:
+      GOVUK_ASSET_ROOT: frontend.dev.gov.uk
+      VIRTUAL_HOST: frontend.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+    command: bin/rails s -P /tmp/rails.pid
+
+  frontend-app-draft:
+    <<: *frontend-app
+    depends_on:
+      - content-store-app-draft
+      - router-app-draft
+      - static-app-draft
+    environment:
+      GOVUK_ASSET_ROOT: draft-frontend.dev.gov.uk
+      VIRTUAL_HOST: draft-frontend.dev.gov.uk
+      PLEK_HOSTNAME_PREFIX: "draft-"
+      HOST: 0.0.0.0
+
+  frontend-app-live:
+    <<: *frontend
```

Comment:
Any reason why this doesn't inherit from frontend-app? we could lose the ports and command if we remove that.

Reply:
I assume because it doesn't need the same dependencies?

Reply:
Done